src/js/searchQuery.js

```diff
@@ -30,7 +30,8 @@ function inputQuerySearch() {
 function movieSearchAPI(page) {
   queryAPI.generalAPI(page).then(value => {   
     results = value.results;
-      addKeyGeners(results);
+    console.log(results);
```

Comment:
remove console.log here, you do not need it

Reply:
remove